sympy/assumptions/ask.py

```diff
@@ -129,6 +131,9 @@ def ask(proposition, assumptions=True, context=global_assumptions):
     else:
         key, expr = Q.is_true, sympify(proposition)
 
+    if isinstance(proposition, (bool, BooleanAtom)):
+        return sympify(proposition)
```

Comment:
Ok. You changed two tests to respect this change. https://github.com/sympy/sympy/pull/2508/files#diff-abc5430bcb3f79fa33a7df36470717fcR1950